### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_ Screen reader: Voiceover is announcing extra information like "1 list item" for 'open folder' button

### ISSUE THREAD

**Issue by kjawale99** [microsoft/vscode#164695]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader.

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor 
2. Tab till "Help" menu bar option .
3. Using "down" arrow key to navigate to "Getting started" and hit enter.
4. Tab navigate till "Open a folder button".
5. Turn on Voiceover
6. Verify that Voiceover is announcing "list" for the button.

### Actual Result:
Voiceover is announcing "list" for the "Open a folder" button.

### Expected Result:
Voiceover should not announce "list" for "Open a folder" button.

### Attachment:
![open a folder](https://user-images.githubusercontent.com/98745784/197939351-0c1e842a-9460-42c7-ad0e-7e8a0a321b75.gif)

**Comment by isidorn** [comment]
Please file against VoiceOver here https://bugs.webkit.org/

**Comment by kjawale99** [comment]
@isidorn Can you please provide the sample app so that we can log third party bug.

**Comment by isidorn** [comment]
@kjawale99 you can use electron fiddle to create an app https://github.com/electron/fiddle

Currently, we on the VS Code team do not create sample app for issue reporters.
